pkg/apis/numaplane/v1alpha1/status_types.go

```diff
@@ -108,7 +108,10 @@ func (s *Status) GetCondition(t ConditionType) *metav1.Condition {
 // Init sets certain Status parameters to a default initial state
 func (status *Status) Init(generation int64) {
 	status.SetObservedGeneration(generation)
-	status.MarkPending()
+	// rationale for commenting this out:
+	// "Pending" is now something we indicate when a rollout has been updated and we are trying to deploy it,
+	// as opposed to meaning that we're "pending reconciliation"
+	//status.MarkPending()
```

Comment:
@afugazzotto wanted to bring this one to your attention as well. I needed to change that.